src/pages/signUp/test/Signup.test.tsx

```diff
@@ -3,10 +3,10 @@ import SignUp from "../SignUp";
 
 describe("<SignUp />", () => {
   it("renders social login buttons", () => {
-    const { queryByLabelText } = renderWithContext(<SignUp />);
+    const { getByLabelText } = renderWithContext(<SignUp />);
 
-    expect(queryByLabelText("google authorization")).toBeTruthy();
-    expect(queryByLabelText("facebook authorization")).toBeTruthy();
-    expect(queryByLabelText("linkedin authorization")).toBeTruthy();
+    expect(getByLabelText("google authorization")).toBeTruthy();
+    expect(getByLabelText("facebook authorization")).toBeTruthy();
+    expect(getByLabelText("linkedin authorization")).toBeTruthy();
```

Comment:
What was the reason for replacing `queryByLabelText` with `getByLabelText`? `queryByLabelText` should work better as it will just make the test fail if the text isn't present, where as `getByLabelText` will throw an error.

Reply:
Well this is interesting... my ts configs say we should be using get instead of query? 🤔
![image](https://user-images.githubusercontent.com/11530120/166617467-5c300d04-3db0-45e6-95ef-09813b207830.png)

Reply:
Interesting, I didn't get that linting error. The libary documentation states that queryBy

> Returns the matching node for a query, and return null if no elements match. This is useful for asserting an element that is not present.

It definitely looks like the linter recommends `getBy` though. I believe that the issue with `getBy` is that the error it throws will cause the tests to end prematurely if the value isn't present, but all of the tests have to pass anyway, so I suppose either way works.

Reply:
I won't modify future tests that have the query but i think it's worth getting our linters in sync sometime